### ISSUE TITLE
Extending Documentation is missing

### ISSUE THREAD

**Issue by JayBox325** [glidejs/glide#386]
Tried to view your Extending Documentation but it's returning a 404.

https://glidejs.com/docs/extending

**Comment by gautamz07** [comment]
@JayBox325  you mean https://glidejs.com/docs/extending-components/ ?

**Comment by JayBox325** [comment]
Yep. The link I used in the official documentation on this page: https://glidejs.com/docs/ links to a 404 so I think this issue is valid.

**Comment by RobertSK** [comment]
He is talking about the link at:

> Extendable
> Need something more custom-made? Plug-in your own modules with additional functionalities.> 
> **Take a look at extending documentation →**

**Comment by JithinAji** [comment]
This issue is still there and in addition I think we should also edit the docs to add an example each for all the available methods.

Is the source code for the [GlideJS website](https://glidejs.com/) included in the source code?
